pyproject.toml

```diff
@@ -30,9 +30,9 @@ dependencies = [
     "scipy>=1.5.2",
     "rustworkx>=0.13.0",
     "qiskit-aer>=0.12.0",
-    "qiskit>=0.45.0, <1.0",
+    "qiskit>=0.46, <2",
```

Comment:
I think this is the best way to specify the version to get us through the 1.0 release. I guess when 1.0 is released we can update the minimum here

Reply:
> I think this is the best way to specify the version to get us through the 1.0 release.

I agree.

Reply:
0.46 does not have package paths we need to fully update to 1.x (e.g. `qiskit.synthesis`), so I think we need to set the qiskit version here to the release candidate.

Reply:
disregard above, I think that's wrong. Need to double check now that fewer things are broken